### PR TITLE
add and test DELETE /api/comments/:comment_id ncnews-10

### PR DESCRIPTION
**Pull request by MichalLewczuk**

### CODE REVIEW

File: __tests__/app.tests.js [hunk 1]
```diff
@@ -438,4 +444,30 @@ describe("COMMENTS TESTS", () => {
       );
     });
   });
+
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("Returns 204 status code for a valid comment id that is in the database", async () => {
```
Comment (dentednerd) [inline __tests__/app.tests.js:449]: Why are you testing for a comment that exists if you're looking at the DELETE method in this block?

Also, the 204 status code would be incorrect for a successful GET request.

File: models/comments.models.js [hunk 2]
```diff
@@ -25,3 +26,14 @@ exports.insertComment = async (article_id, username, body) => {
 
   return result.rows[0];
 };
+
+// REMOVE
+
+exports.removeCommentById = async (comment_id) => {
+  const result = await db.query(
+    "DELETE FROM comments WHERE comment_id = $1 RETURNING *;",
```
Comment (dentednerd) [inline models/comments.models.js:34]: Would there be anything to return from this query if something has been deleted?

**Review by dentednerd — CHANGES_REQUESTED** [review]
Michal, I can see that in a previous ticket, you've created a `helpers` directory in `db`. That's what `utils` is for! Utility functions === helper functions. I can also see that the function you created, `checkExists`, hasn't been tested. I'd like to see some tests for it before I can approve this PR. You also need to reconsider your tests around the DELETE endpoint, but please test checkExists as a priority.

File: __tests__/app.tests.js (hunk 1 above)
Comment (MichalLewczuk) [reply __tests__/app.tests.js:449]: The 204 is for the delete DELETE.
The GET requests are to check articles endpoint, the first checks if there is no longer a comment with that id, and the second checks the count of comments after deletion.

File: models/comments.models.js (hunk 2 above)
Comment (MichalLewczuk) [reply models/comments.models.js:34]: The returning does return the deleted comment, though I'm not using it for anything at the moment.

**Pull request closed without merging**